### PR TITLE
Using a plugin with chartist in Ember

### PR DESCRIPTION
**Pull request by jewls618**
This solves #31

### CODE REVIEW

**Comment by jherdman** [comment]
Ooh. Very interesting. I'll give this a try ASAP.

File: index.js [hunk 1]
```diff
@@ -29,5 +29,11 @@ module.exports = {
     }
 
     app.import(path.join(chartistPath, 'chartist.js'));
+     if(options.plugins){
```
Comment (jherdman) [inline index.js]: Would you mind fixing the indentation here?

File: README.md [hunk 2]
```diff
@@ -163,6 +163,18 @@ you can also import the Chartist settings scss:
...
 
+## Plugins
+
+Install the plugin with Bower.
+
+In ember-cli-build.js, use the following example to add the plugin name: 
+
+```
+		'ember-cli-chartist': {
+			plugins: ['chartist-plugin-legend']
```
Comment (jherdman) [inline README.md]: Something we all basically know that's going to happen is that Bower go away (or at least be much less used). I wonder if we should future-proof this API a bit? Something more like:

```javascript
plugins: [
  {
    name: 'chartist-plugin-legend',
    type: 'bower', /* we could even assume this by default? */
     /* possible other options, e.g. override the name used to search the bower dir */
  }
],
```

**Comment by jewls618** [comment]
Ended up installing via npm so we could have the latest version!

**Comment by jherdman** [comment]
> Ended up installing via npm so we could have the latest version!

Would you mind elaborating? Is there something this addon could be doing to help you out?

**Comment by alechirsch** [comment]
@jherdman To get it to work we installed this package `ember-cli-node-modules-to-vendor`, which allowed us to include the charist plugin in ember-cli-build.js

```
let app = new EmberApp(defaults, {
	nodeModulesToVendor: [
		'node_modules/chartist-plugin-legend'
	]
});

app.import('vendor/chartist-plugin-legend.js');
```

One of the problems with making it generic would be to know where the main file is in the node module. This addon could help by allowing the user to configure the chartist plugins that they would want to use in an easy way.

I am not clear on how this can be achieved using npm packages, ember makes it very easy to include bower packages which is why we initially took that approach. We ran into a problem where the bower package was at least 5 minor releases behind the npm package and did not have the features that we needed.

**Pull request closed without merging**